### ISSUE TITLE
Format File for Invoke-SqlAssessment of SQL Server Assessment APIs

### ISSUE THREAD

**Issue by shaneis** [microsoft/sql-server-samples#607]
The cmdlet `Get-SqlAssessment` has a format file associated with it (_I can't find it in SQLProvider.Format.ps1xml but it is getting formatted_).

![image](https://user-images.githubusercontent.com/8527247/62857650-1317a380-bcf0-11e9-8210-87f3ca2e914e.png)

It's a great help showing only the information that is needed at a glance and saving the extra information in case it'll be used down the pipeline, like `Level`, `Tags`, etc.

![image](https://user-images.githubusercontent.com/8527247/62857759-4fe39a80-bcf0-11e9-87ab-78281f4ca46e.png)

# Invoke-SqlAssessment

`Invoke-SqlAssessment` does not have a format file and I'd argue that it would require one.
The expectation would be that the results of the `Invoke-SqlAssessment` would be looked at more than people would look at the results of `Get-SqlAssessmentItem`

## Current View

Currently the cmdlet returns : 

### Default

![image](https://user-images.githubusercontent.com/8527247/62858050-2b3bf280-bcf1-11e9-9409-125be94165c8.png)

### With Select *

![image](https://user-images.githubusercontent.com/8527247/62858192-91287a00-bcf1-11e9-8cd7-1d3332c89f62.png)

which are the exact same data.

However, I'd argue that the information required at a glance would be properties : `CheckId`, `CheckName`, `Message`, and (potentially) `HelpLink`.

# Action

Can we include a format file for `Invoke-SqlAssessment` that returns the properties  : `CheckId`, `CheckName`, `Message`, and (potentially) `HelpLink`?

![image](https://user-images.githubusercontent.com/8527247/62858795-35f78700-bcf3-11e9-9c4d-9a6b00e97558.png)

**Comment by aguzev** [comment]
Thanks Shane,
How much is the check name useful in the `Invoke-SqlAssessment` output? What about `CheckId`, `Severity`, `Message` format, where Severity is **Information**, **Warning**, or **Critical**? 

I believe, Message is the most important part of the output, so It should get more space than others.

**Comment by shaneis** [comment]
Makes sense for me. Since the `CheckId` is fairly readable, it can take the place of the `CheckName` 😄 

Some of the messages can get pretty large e.g. `CheckId    : SqlServer.Database.ForeignKeysNoIndices` so giving `Message` more space is a good idea

**Comment by aguzev** [comment]
In case we made check ids shorter and more cryptic, would you still need check names in the Invoke-SqlAssessment output? How could you use the name in the next command?

**Comment by shaneis** [comment]
Right now, I'm thinking that the check names would be used for reporting / grouping / filtering.

Whether that's reporting out in a dashboard or grouping / filtering out by `CheckName` e.g databases without Query Store enabled.
This could happen in situations where changes have to be discussed and reviewed by a Change Approval Group before they can take place.

```PowerShell
Invoke-SqlAssessment -InputObject $Server.Databases |
    Where-Object CheckName -eq 'Query Store should be on'
```

I'd use `Message` as a way of grouping them out but some messages have custom output like `ForeignKeysNoIndices` that would need some manipulation to have them match up.
And if the `CheckId`s could become shorter and more cryptic, then `CheckName` is what you'd use to group them.
